extensions/amp-stream-gallery/1.0/component.js

```diff
@@ -40,15 +41,15 @@ function BentoStreamGalleryWithRef(props, ref) {
   const classes = useStyles();
   const carouselRef = useRef(null);
   const [visibleCount, setVisibleCount] = useState(DEFAULT_VISIBLE_COUNT);
-
+  const length = useMemo(() => toChildArray(children), [children]);
```

Comment:
nit: `useMemo` might be an overkill here. Typically its used for expensive calculations - calculating length isn't one of them 🙂

Reply:
Good catch on both accounts, thanks.